### ISSUE TITLE
Admin controls

### ISSUE THREAD

**Issue by calxus** [kaffo/multi-device-audio-project#16]
Delete functionality on the back end, with a button to do so on the front end.

Ties in with #8.

On the user page just have a box with "admin controls".

Standard user can use these settings to delete their own recordings. Admin users can delete all.

**Comment by kaffo** [comment]
Will deal with this after #8

**Comment by kaffo** [comment]
Luckily for us django already has a build in "superuser" flag for users. I think it's fair to say superusers can be our admins for the time being. I could get this working in about 5 minutes but it wouldn't be very nice. I'll speak to the team tomorrow.

**Comment by ppyordanov** [comment]
Doesn't the admin.py file do what we want?

**Comment by allyjweir** [comment]
Need an admin page to show if the user is an admin still. Anyone on this? @Kaffse?

**Comment by kaffo** [comment]
Oh darn, I forgot this was still an issue. I'll go look into it. If I'm right it shouldn't be a big deal...probably.

**Comment by allyjweir** [comment]
Could these controls not be dumped into that settings page we just got rid of?
- Add avatar
- Change email
- Change password
- Delete account
- ...
- Profit?

**Comment by kaffo** [comment]
I think it was dealing more with the management of recordings more than the user account. I've got code ready for deleting stuff from the database and code to detect if the user is an admin. Just need the actual recordings displayed correctly to add the delete button as such. 
As for user's deleting their own stuff... I've done a lot of thinking on it and I can't come up with a solution that isn't either A. overcomplicated or B. exploitable thanks to the fact we are dealing with it on the front end. 
At the end of the day this isn't a needed feature, but it'd be nice to have. I can talk about what I mean in the meeting today.
